.github/workflows/build.yml

```diff
@@ -59,6 +59,25 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-go-
 
+    - name: Test
+      run: make test
+      working-directory: amazon-cloudwatch-agent
+
+    - name: Upload coverage to Codecov
```

Comment:
I don't think that we need to/should upload test coverage on a pull request. Shouldn't that only be when we merge to master?

Reply:
It's not setup yet, but it will eventually allow for blocking merge if test coverage goes down.

Reply:
Works for me.